kubevirt/schema/datavolume/data_volume.go

```diff
@@ -3,16 +3,16 @@ package datavolume
 import (
 	"fmt"
 
-	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/kubevirt/terraform-provider-kubevirt/kubevirt/schema/k8s"
 	"github.com/kubevirt/terraform-provider-kubevirt/kubevirt/utils/patch"
-	cdiv1 "kubevirt.io/containerized-data-importer/pkg/apis/core/v1alpha1"
+	cdiv1 "kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1"
 )
 
 func DataVolumeFields() map[string]*schema.Schema {
 	return map[string]*schema.Schema{
 		"metadata": k8s.NamespacedMetadataSchema("DataVolume", false),
-		"spec":     dataVolumeSpecSchema(),
+		"spec":     DataVolumeSpecSchema(),
 		"status":   dataVolumeStatusSchema(),
 	}
 }
```

Comment:
I saw you added DataVolumeTemplatesSchema to be part of virtualmachine package - 
Does DataVolumeTemplatesSchema (line 20) still in use?

Reply:
You right, its not used anymore, removed it